Plot/Components/PartialComponents/Loader/Loader.razor

```diff
@@ -0,0 +1,37 @@
+<!-- Filename: Loader.razor
+Part of Project:-- PLOT
+File Purpose:
+Create a component to display a loading symbol for when the
+program is loading
+
+Program Purpose:
+The purpose of PLOT is to allow users to easily create, manage, 
+and allocate floorsets for Platos Closet. 
+
+Author: Josh Rodack (2/10/2025)-->
+
+
+
+            <script src="https://kit.fontawesome.com/b9b256c8f6.js" crossorigin="anonymous"></script>
+            
+            <div class="Loader @Class" id="@Id">
+            <div class="fa-3x">
+            <i class="fa-solid fa-spinner fa-spin-pulse"></i>
+            </div>
+            <label>
+                @LabelText
+            </label>
+            </div>
+            
+
+@code {
+
+    [Parameter]
+    public string LabelText { get; set;}
```

Comment:
Also @JoshRodack, if you could change this to be a `RenderFragment` instead. It should allow other devs to put the text between the `<Loader>Text</Loader>` tags instead of having to put it in as a attribute. For more information about `RenderFragment` you can use this link:

https://learn.microsoft.com/en-us/aspnet/core/blazor/components/?view=aspnetcore-9.0#child-content-render-fragments